pkgs/top-level/all-packages.nix

```diff
@@ -12134,7 +12134,22 @@ in
 
   dbxml = callPackage ../development/libraries/dbxml { };
 
-  dbus = callPackage ../development/libraries/dbus { };
+  dbus = callPackage ../development/libraries/dbus { systemd = systemd.override {
+      withResolved = false;
+      withLogind = false;
+      withHostnamed = false;
+      withLocaled = false;
+      withTimedated = false;
+      withHwdb = false;
+      withEfi = false;
+      withImportd = false;
+      withHomed = false;
+      withPortabled = false;
+      dbus = null;
+      cryptsetup = null;
+      lvm2 = null;
+    };
```

Comment:
Huuuh, so we pull in another systemd here, to bootstrap dbus, which is being used by systemd?

Did you check if it'll become a runtime dependency?

I'm not sure if this is the right path forward - also see https://github.com/NixOS/nixpkgs/issues/98094 for another bootstrap systemd leaking in :-/

Reply:
It's a runtime dependency, yes. Maybe we could patchelf dbus to search in /run/current-system for libsystemd

Reply:
Systemd's DBUS dependency is _only_ used in its test suite (which we disable); so we should be able to just set `dbus = null` on our toplevel `systemd` instead of providing a `systemdMinimal` 

Source:

https://github.com/systemd/systemd/blob/master/meson_options.txt#L338-L339